cpp/velox/compute/VeloxBackend.cc

```diff
@@ -251,8 +251,6 @@ void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf
     facebook::velox::serializer::presto::PrestoVectorSerde::registerVectorSerde();
   }
   velox::exec::Operator::registerOperator(std::make_unique<RowVectorStreamOperatorTranslator>());
-
-  initUdf(conf);
```

Comment:
@marin-ma Would you please take a look at the change? Thanks!

Reply:
UDF related UT has passed, any comments?

Reply:
@marin-ma could you help confirm that?

Reply:
@Yohahaha Could you follow the comment and run this suite? It failed in my local environment. https://github.com/oap-project/gluten/blob/00f83a40e753e3b33d8e9a38df40b48d22b8416e/backends-velox/src/test/scala/io/glutenproject/expression/VeloxUdfSuite.scala#L89-L100

Reply:
I revert changes of initUdf logic, it works.

The logic of load/register udf confused me lot...